package.json

```diff
@@ -26,30 +34,44 @@
     "react": ">=16.3.0 <19"
   },
   "jest": {
-    "preset": "amex-jest-preset-react"
+    "preset": "amex-jest-preset-react",
```

Comment:
This is not neccissary, but you can feel free to convert this repo to use `vitest` as well if you like.

But if you dont want to during this change, feel free not to